src/main/java/uk/gov/justice/digital/hmpps/whereabouts/config/ResourceServerConfiguration.java

```diff
@@ -42,6 +42,7 @@ public ResourceServerConfiguration(@Autowired(required = false) final BuildPrope
     @Override
     public void configure(final HttpSecurity http) throws Exception {
         http
+                .headers().frameOptions().sameOrigin().and()
```

Comment:
allow content to be served in iframes if in same origin. 

To fix access to h2 console in dev mode.